.editorconfig

```diff
@@ -51,3 +51,12 @@ dotnet_style_explicit_tuple_names                          = true : suggestion
 
 trim_trailing_whitespace = true
 insert_final_newline = true
+
+# Suggest to use "_" instead of "this." when creating new private fields
+dotnet_naming_style.camel_case_leading_underscore.capitalization = camel_case
+dotnet_naming_style.camel_case_leading_underscore.required_prefix = _
+dotnet_naming_symbols.private_fields.applicable_kinds = field
+dotnet_naming_symbols.private_fields.applicable_accessibilities = private
+dotnet_naming_rule.private_fields_should_be_camel_case_leading_underscore.severity = suggestion
+dotnet_naming_rule.private_fields_should_be_camel_case_leading_underscore.symbols = private_fields
+dotnet_naming_rule.private_fields_should_be_camel_case_leading_underscore.style = camel_case_leading_underscore
```

Comment:
add a new line all the time at the end of your files for Git.

Reply:
done